doc/Tests_XML_Definition_and_Sample.md

```diff
@@ -33,6 +44,186 @@ Example:
+<?xml-stylesheet type="text/xsl" href="./testcase.xsl"?>
+<test_definition>
+  <suite category="Runtime" name="wrt-rt-linux-tests">
+  ...
+    <set name="connectsrccrossoriginmultixmlhttprequestallowedtwo" type="wrt">
+     <testcase component="Crosswalk WRT/CSP" execution_type="auto" id="connectsrccrossoriginmultixmlhttprequestallowedtwo" priority="P1" purpose="Check if user agent is able to open second allowed external resource by xhr when connect-src is cross-origin." status="approved" type="compliance">
+        <description>
+          <pre_condition/>
+          <post_condition/>
+          <steps>
+            <step order="1">
+              <step_desc>Check if user agent is able to open second allowed external resource by xhr when connect-src is cross-origin.</step_desc>
+              <expected>PASS</expected>
+            </step>
+          </steps>
+          <test_script_entry test_script_expected_result="0">test_index.html</test_script_entry>
+        </description>
+      </testcase>
+    </set>
+  </suite>
+</test_definition>
+```
+
+#### "qunit" type
+
+The qunit style test
+
+Example:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="./testcase.xsl"?>
+<test_definition>
+  <suite category="SIMD" name="webapi-simd-nonw3c-tests">
+    <set name="webapi-simd-nonw3c-tests" type="qunit">
+      <testcase component="Supplementary APIs/Experimental/SIMD" execution_type="auto" id="Float32x4_constructor" priority="P1" purpose="Check Float32x4 constructor (0, 2, 2)" status="approved" subcase="6" type="compliance">
+        <description>
+          <test_script_entry>/opt/webapi-simd-nonw3c-tests/simd/ecmascript_simd/src/index.html?testNumber=1</test_script_entry>
+        </description>
+        <specs>
+          <spec>
+            <spec_assertion category="SIMD" element_name="Float32x4" element_type="method" interface="SIMD" section="SIMD" specification="SIMD"/>
+            <spec_url>https://github.com/johnmccutchan/ecmascript_simd/blob/master/README.md</spec_url>
+            <spec_statement/>
+          </spec>
+        </specs>
+      </testcase>
+    </set>
+  </suite>
+</test_definition>
+```
+#### "nodeunit" type
+
+The test load with node js unit test runner
+
+Example:
+
+```xml
+
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="./testcase.xsl"?>
+<test_definition>
+    <set name="common_module" type="nodeunit">
+      <testcase component="Crosswalk App Tools/CLI" execution_type="auto" id="Crosswalk_create_no_sdk" priority="P2" purpose="Android - Validate if project is created fail without android sdk" status="approved" type="Functional">
+        <description>
+          <test_script_entry>/opt/apptools-android-tests/apptools/create_no_sdk.py</test_script_entry>
+        </description>
+      </testcase>
+    </set>
+  </suite>
+</test_definition>
+```
+###Core Test Type 
+
+#### "script" type
+
+The test case with directly script execution entry
```

Comment:
=> This type of tests will let testkit-lite directly execute the script in `<test_script_entry>`.